includes/theme-integrations/theme-integration-loader.php

```diff
@@ -79,6 +79,14 @@ private function possibly_load_supported_theme_wrappers(){
             $supported_theme_class_file = trailingslashit( Sensei()->plugin_path ) . 'includes/theme-integrations/' . $this->active_theme . '.php';
             $supported_theme_class_name  = 'Sensei_'. ucfirst( $this->active_theme  );
 
+            // add name of theme as body class for detailed styling.
+            add_filter( 'body_class', 'sensei_current_theme' );
+            function sensei_current_theme( $classes ) {
```

Comment:
It might be best to move that function outside of `possibly_load_supported_theme_wrappers`. You can keep it in the class, but in its own function.

Since it's in the class, you don't need to use the `sensei` prefix in the function name here.